### ISSUE TITLE
CursorLine doesn't override virtual text background

### ISSUE THREAD

**Issue by d-miketa** [theHamsta/nvim-dap-virtual-text#17]
<img width="206" alt="image" src="https://user-images.githubusercontent.com/320321/132897112-46437b95-f07c-40a7-8ee1-22bfe18ab089.png">
I'd expect `CursorLine` background under the virtual text.

I've tried changing `NvimDapVirtualText guibg=NONE` etc but it didn't work. :( `Comment` doesn't have this issue (the background respects `CursorLine`) and `NvimDapVirtualText` still links to it.

**Comment by theHamsta** [comment]
Hmm, that changed quite recently for most virtual text. Did you see a plugin with virtual text where this does not happen? Maybe it's an upstream issue.

**Comment by d-miketa** [comment]
No, but I don't use any other virtual text plugins, sorry. :/

**Comment by theHamsta** [comment]
Maybe you can create an upstream issue? This is the whole code that you need to reproduce.
```
vim.api.nvim_buf_set_virtual_text(
	vim.api.nvim_get_current_buf(),
	0,
	0,
	{ { "hello", "Comment" } },
	{}
)
```

**Comment by theHamsta** [comment]
Fixed by #32
